requirements/prod.txt

```diff
@@ -77,7 +75,6 @@ pytz==2013.8
 raven==4.2.3
 rdflib==3.0.0
 recaptcha-client==1.0.6
-receipts==0.2.9
```

Comment:
Not related to the bug, but I believe it uses browserid. Also, we don't use it on AMO, it's a leftover from the marketplace before the guillotine.